test/unit/util.test.ts

```diff
@@ -1,4 +1,5 @@
 import { JSDOM } from "jsdom"
+import { Cookie as CookieEnum } from "../../lib/vscode/src/vs/server/common/cookie"
```

Comment:
Importing this from `lib/vscode` breaks the expected directory structure in `out`, see https://github.com/cdr/code-server/pull/2922#issuecomment-801746756